code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -52,7 +60,7 @@ var/global/list/parasites = list() //all currently existing/living guardians
 
 /mob/living/simple_animal/hostile/guardian/proc/setthemename(pickedtheme) //set the guardian's theme to something cool!
 	if(!pickedtheme)
-		pickedtheme = pick("magic", "tech")
+		pickedtheme = pick("magic", "tech", "carp")
```

Comment:
It's not necessary but this pick shouldn't be called anyway; it should always be getting a specific theme unless you're doing testing and have disabled dying without a summoner